app/src/main/java/com/example/grutor/Fragments/RegisterFragment.java

```diff
@@ -1,16 +1,31 @@
 package com.example.grutor.Fragments;
 
+import android.content.Intent;
 import android.os.Bundle;
 
 import androidx.fragment.app.Fragment;
 
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Spinner;
 
+import com.example.grutor.Activites.FeedActivity;
 import com.example.grutor.R;
+import com.parse.ParseUser;
 
 public class RegisterFragment extends Fragment {
+    public  Spinner spGrades;
+    public Spinner spSubjects;
+    public Boolean gradeSelect = false;
+    public Boolean subjectSelect = false;
```

Comment:
If these variable are not accessed outside of this class, please make them to private. it is risky to access UI element outside of fragment.